solo-chains/runtime/starlight/src/tests/relay_registrar.rs

```diff
@@ -95,3 +100,505 @@ fn registrar_needs_a_reserved_para_id() {
                 .is_parathread());
         });
 }
+
+#[test]
+fn register_para_via_container_registrar() {
+    ExtBuilder::default()
+        .with_para_ids(vec![
+            (1001, empty_genesis_data(), u32::MAX, u32::MAX).into(),
+            (1002, empty_genesis_data(), u32::MAX, u32::MAX).into(),
+        ])
+        .build()
+        .execute_with(|| {
+            // In this test we're gonna register a para via ContainerRegistrar,
+            // which will internally use the InnerRegistrar type to also register the para
+            // in the relay Registrar pallet.
+
+            assert_eq!(Runtime::registered_paras(), vec![1001.into(), 1002.into()]);
+            assert_eq!(Runtime::genesis_data(1003.into()).as_ref(), None);
+            run_to_session(1u32);
+
+            // Change max_head_data_size config.
+            assert_ok!(
+                Configuration::set_max_head_data_size(root_origin(), 20500),
+                ()
```

Comment:
Is this needed? Because I see you also set it in genesis

Reply:
Nop, this is not needed anymore, as it's being already set in genesis. 

Fixed in 73daab1933b5cfc51de53b7c7e8512420e7907c0.